sql/moz-fx-data-shared-prod/monitoring_derived/structured_missing_columns_v1/query.py

```diff
@@ -58,16 +59,26 @@
     document_type,
     document_version,
     path
+),
+
+information_schema_col AS (
+  SELECT field_path  AS column_name_in_table
+  FROM 
+    `moz-fx-data-shared-prod.{namespace}_stable.INFORMATION_SCHEMA.COLUMN_FIELD_PATHS`
```

Comment:
One issue with this approach:

We'd like to only show missing columns in the Looker dashboard that don't have an entry in `INFORMATION_SCHEMA.COLUMN_FIELD_PATHS`. So if a column `a` showed up as missing on say 2023-01-01 but was added on 2022-01-02 the Looker dashboard would still show the column as missing for 2023-01-01 (but not for 2022-01-02) since this table is only updated daily. We'd ideally not show column `a` as missing after 2022-01-02 since we know through `INFORMATION_SCHEMA.COLUMN_FIELD_PATHS` that it got added.

I wondering if we need to move the check on `INFORMATION_SCHEMA.COLUMN_FIELD_PATHS` to the lookml instead.